### PR TITLE
Added new menthod to demo class

### PR DESCRIPTION
**Pull request by krishnaagupta**
Added a method in demo class

### CODE REVIEW

File: pageObjectModel/src/test/java/TestCases/demo.java [hunk 1]
```diff
@@ -5,5 +5,7 @@
 	public demo() {
 		// TODO Auto-generated constructor stub
 	}
-
+	public void demo2() {
+		// TODO Auto-generated constructor stub
```
Comment (sunigup2) [inline pageObjectModel/src/test/java/TestCases/demo.java:9]: Add logic to the method

**Review by sunigup2 — CHANGES_REQUESTED** [review]
Please address review comments

File: pageObjectModel/src/test/java/TestCases/demo.java (hunk 1 above)
Comment (krishnaagupta) [reply pageObjectModel/src/test/java/TestCases/demo.java:9]: done

**Review by sunigup2 — APPROVED** [review]
LGTM

**Pull request merged**